packages/aws-cdk/README.md

```diff
@@ -971,6 +971,22 @@ cdk gc --unstable=gc --action=delete-tagged --rollback-buffer-days=30
 
 This will delete assets that have been unused for >30 days, but will not tag additional assets.
 
+Here is a diagram that shows the algorithm of garbage collection:
+
+![Diagram of Garbage Collection algorithm](/packages/aws-cdk/images/garbage-collection.png)
+
+#### Theoretical Race Condition with `REVIEW_IN_PROGRESS` stacks
```

Comment:
It's not just this; it's also stacks for stacks created from nothing. We could have:

```
(Assets uploaded) --> (GC runs) --> (Stack created)
```

REVIEW_IN_PROGRESS is a subphase of stacks being created, but the race condition applies regardless.

Mention the mitigation on our end ("created buffer days"), and include a recommendation to not deploy stacks and run GC at the same time. The workaround (redeploy) and "report an issue" are great callouts!